modules/library/cql/src/main/java/org/geotools/filter/text/commons/ExpressionToText.java

```diff
@@ -184,17 +186,40 @@ else if (literal instanceof Period){
             
             output = dateToText( period.getBeginning().getPosition().getDate(), output );
             output.append("/");
-    		output = dateToText( period.getEnding().getPosition().getDate(), output );
-    		
-    		return output;
+            output = dateToText(period.getEnding().getPosition().getDate(), output);
+
+            return output;
+        }
+        else if (literal instanceof Color){
+            Color color = (Color) literal;
+            
+            String redCode = Integer.toHexString(color.getRed());
+            String greenCode = Integer.toHexString(color.getGreen());
+            String blueCode = Integer.toHexString(color.getBlue());
```

Comment:
I duplicating what the color converter does exactly to prevent surprises, alpha is usually handled by a seperate opacity setting after all.